app/models/question.rb

```diff
@@ -40,66 +47,68 @@ class Question < ApplicationRecord
-    validates :"casted_#{field}", numericality: { allow_blank: true, greater_than: -1e7, less_than: 1e7 }
+    validates :"casted_#{field}", numericality: {allow_blank: true, greater_than: -1e7, less_than: 1e7}
   end
   validate :code_unique_per_mission
   validate :at_least_one_name
   validate :valid_reference_url
 
-  scope :by_code, -> { order('questions.code') }
+  scope :by_code, -> { order("questions.code") }
   scope :with_code, ->(c) { where("LOWER(code) = ?", c.downcase) }
   scope :default_order, -> { by_code }
-  scope :select_types, -> { where(:qtype_name => %w(select_one select_multiple)) }
+  scope :select_types, -> { where(qtype_name: %w[select_one select_multiple]) }
   scope :with_forms, -> { includes(:forms) }
-  scope :reportable, -> { where.not(qtype_name: %w(image annotated_image signature sketch audio video)) }
-  scope :not_in_form, ->(form) { where("(questions.id NOT IN (
-    SELECT question_id FROM form_items
-      WHERE type = 'Questioning' AND form_id = ? AND deleted_at IS NULL))", form.id) }
+  scope :reportable, -> { where.not(qtype_name: %w[image annotated_image signature sketch audio video]) }
+  scope :not_in_form, -> (form) {
+                        where("(questions.id NOT IN (
+                          SELECT question_id FROM form_items
+                            WHERE type = 'Questioning' AND form_id = ? AND deleted_at IS NULL))", form.id)
+                      }
 
   translates :name, :hint
 
   delegate :smsable?,
-           :has_options?,
-           :temporal?,
-           :numeric?,
-           :printable?,
-           :multimedia?,
-           :odk_tag,
-           :odk_name,
-           :select_multiple?,
-           to: :qtype
+    :has_options?,
+    :temporal?,
+    :numeric?,
+    :printable?,
+    :multimedia?,
+    :odk_tag,
+    :odk_name,
+    :select_multiple?,
+    to: :qtype
 
   delegate :options,
-           :first_level_option_nodes,
-           :all_options,
-           :first_leaf_option,
-           :first_leaf_option_node,
-           :first_level_options,
-           :multilevel?,
-           :level_count,
-           :level,
-           :levels,
-           :sms_formatting_as_text?,
-           :sms_formatting_as_appendix?,
-           to: :option_set, allow_nil: true
-
-  replicable child_assocs: :option_set, backwards_assocs: :questioning, sync: :code,
-    uniqueness: {field: :code, style: :camel_case}, dont_copy: [:key, :access_level, :option_set_id],
-    compatibility: %i[qtype_name option_set_id], reusable_in_clone: true
+    :first_level_option_nodes,
+    :all_options,
+    :first_leaf_option,
+    :first_leaf_option_node,
+    :first_level_options,
+    :multilevel?,
+    :level_count,
+    :level,
+    :levels,
+    :sms_formatting_as_text?,
+    :sms_formatting_as_appendix?,
+    to: :option_set, allow_nil: true
+
+  replicable child_assocs: %i[option_set taggings], backwards_assocs: :questioning, sync: :code,
```

Comment:
`replicable` declaration updated to add `taggings`. all other changes in this file are rubocop fixes.